database/factories/CommentFactory.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Database\Factories;
+
+use App\Community\Enums\ArticleType;
+use App\Models\Comment;
+use Illuminate\Database\Eloquent\Factories\Factory;
+
+/**
+ * @extends Factory<Comment>
+ */
+class CommentFactory extends Factory
+{
+    protected $model = Comment::class;
+
+    public function definition(): array
+    {
+        return [
+            'Payload' => $this->faker->paragraph,
+            'Submitted' => $this->faker->dateTimeBetween('-1 year', 'now'),
+            'Edited' => $this->faker->dateTimeBetween('now', '+1 year'),
```

Comment:
There should be a high likelihood of Edited being null, and I'm not sure what unexpected impacts future dates would have.

Reply:
Agreed. In latest, this field now only has a one in twelve chance of being populated.